data-model/fdo-profiles/0.1.0/tettris-service/schema/tettris-service.json

```diff
@@ -3,37 +3,145 @@
+      "example": "https://hdl.handle.net/10.17183",
+      "$comment": "idx = 6"
+    },
+    "pidIssuerName": {
+      "type": "string",
+      "description": "Human-readable name of the PID issuer",
+      "example": "DataCite",
+      "$comment": "idx = 7"
+    },
+    "issuedForAgent": {
+      "type": "string",
+      "description": "In the case of a digital specimen, this is a PID for DiSSCo as the agent responsible for serving the digital specimen object",
+      "example": "https://hdl.handle.net/10.22",
+      "$comment": "idx = 8"
+    },
+    "issuedForAgentName": {
+      "type": "string",
+      "description": "Human-readable agent name",
+      "example": "DiSSCo",
+      "$comment": "idx = 9"
+    },
+    "pidRecordIssueDate": {
+      "type": "string",
+      "description": "Date the PID record was created",
+      "example": "2023-10-16T12:08:37.494Z",
+      "$comment": "idx = 10"
+    },
+    "pidRecordIssueNumber": {
+      "type": "integer",
+      "description": "Starts with 1 and is incrementally increased by 1 every time the pid record is updated. Compatible with DOI schema requirements.",
+      "example": "2",
+      "$comment": "idx = 11"
+    },
+    "structuralType": {
+      "enum": [
+        "digital",
+        "physical",
+        "performance",
+        "abstraction"
+      ],
+      "description": "Nature of the digital object, compatible with DOI schema requirements. The nature of a digital specimen object is always \"digital\". Other digital objects (outside DiSSCo) could be of physical, performance or abstraction nature.",
+      "$comment": "idx = 12"
+    },
+    "pidStatus": {
+      "enum": [
+        "DRAFT",
+        "ACTIVE",
+        "RETIRED",
+        "OBSOLETE",
+        "FAILED",
+        "MERGED",
+        "SPLIT",
+        "TEST"
+      ],
+      "description": "A PID is considered to have a lifecycle, PID status indicates the status in the life cycle, e.g. draft, active, retired. PID statuses are described further in the PID infrastructure design.",
+      "$comment": "idx = 13"
+    },
+    "HS_ADMIN": {
+      "type": "string",
+      "description": "Administrative information for PID profile",
+      "$comment": "idx = 100"
+    },
+    "10320/loc": {
+      "type": "string",
+      "description": "XML document describing location(s) of the resource",
+      "$comment": "idx = 101",
+      "example": "<locations><location href=\"https://dev.dissco.tech/api/v1/annotations/TEST/FFR-RTL-B8K\" id=\"0\" weight=\"1\"/></locations>"
+    },
+    "resourceType": {
+      "enum": [
+        "Digital",
+        "Physical",
+        "Expertise"
+      ],
+      "$comment": "idx = 350"
+    },
+    "serviceCategory": {
+      "enum": [
+        "Reference collections",
+        "Training datasets",
+        "R-packages",
+        "Identification tools",
+        "MAS",
+        "Lab tools",
+        "Crowd sourcing tools for digitisation",
+        "Training platforms (DEST)"
+      ],
```

Comment:
@wouteraddink we have to update this when we create the subset of categories from the EOSC marketplace